event-sourcing/event-repository/event-repository-jdbc/src/test/java/uk/gov/justice/services/eventsourcing/repository/jdbc/eventlog/EventLogConverterTest.java

```diff
@@ -90,6 +93,7 @@ private JsonEnvelope createTestEnvelope() throws IOException {
         final Metadata metadata = metadataFrom(Json.createObjectBuilder()
                 .add("id", ID.toString())
                 .add("name", NAME)
+                .add(DATE_CREATED, ZonedDateTimes.toString(clock.now()))
```

Comment:
Looks like we're not testing metadata here - we should add tests to cover this. Probably need to use the StoppedClock for that.